### ISSUE TITLE
Parse and format env variable before storage

### ISSUE THREAD

**Issue by ccourson** [rangav/thunder-client-support#710]
A login request I perform returns a long authorization token as a text value between double quotes ("). The value cannot be used with the quotes so I have to remove the quotes from the env variable before firing any other request using the token.

Maybe a way to specify the value using regex in a setTo Test? i.e. {{variable/regex/, env}}
Another idea might be to add a term to remove quotes sort of like Jinja filters: {{variable|no-quotes, env}}

Thank you for your time. The tool is awesome.

**Comment by rangav** [comment]
Thanks @ccourson for the feedback, will add to roadmap.

**Comment by rangav** [comment]
looks like a cool idea, might suit this use-case and other common operations, will look into it more when I got time. Thanks for suggestion.

**Comment by qbasic16** [comment]
> Thanks @ccourson for the feedback, will add to roadmap.

Hi rangav,

I have experienced the same problem. Here is some additional information:
I use the OAuth2 "Authorization Code with PKCE" and my linked ".env" file has the values encosed in double quotes `"` just like ccourson stated.

When I press "Generate Token", the URL in the request turns out as follows:
(see the `%22` for the query param `client_id`)

```
https://xxxxxx.b2clogin.com/xxxxxx.onmicrosoft.com/oauth2/v2.0/authorize?p=b2c_1_signinsignup&response_type=code&state=&client_id=%22xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx%22&scope=https://xxxxxx.onmicrosoft.com/api/Read%20offline_access&redirect_uri=https://www.thunderclient.com/oauth/callback&code_challenge=xxxxxx&code_challenge_method=S256
```

My `.env` looks like this:

```dotenv
AUTH_SCOPE_READ="https://xxxxxx.onmicrosoft.com/api/Read"
AUTH_AUTHORITY_SIGNIN_SIGNUP="https://xxxxxx.b2clogin.com/xxxxxx.onmicrosoft.com/B2C_1_SignInSignUp"
AUTH_CLIENT_ID="xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
AUTH_CLIENT_SECRET="xxxxxxxxxxxxxxxx"
AUTH_REDIRECT_URI="http://localhost:3000/auth/redirect"
```

I'm not familiar with VS Code Extensions yet, but if you can use NPM packages, I can recommend [dotenv](https://www.npmjs.com/package/dotenv) to use for parsing so it will adhere to common formatting.

Thank you for this awesome extension!
Cheers
Peter

**Comment by rangav** [comment]
@qbasic16 I have update the library to `dotenv` to read .env files in `v1.19.3`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v1.19.3

**Comment by rangav** [comment]
This feature is now implemented and published to marketplace, please update to `v2.0.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.0.0

Please let me know your feedback.

@ccourson thanks for suggesting jinja filters, its now implemented in extension.

Remove Quotes
<img width="757" alt="Screenshot 2022-11-17 at 11 46 10" src="https://user-images.githubusercontent.com/8637550/202438159-586f5eb6-4552-4c17-a1e9-44524338044b.png">
